lib/bumper.js

```diff
@@ -207,27 +203,37 @@ class Bumper {
 
   /**
    * Get the PR scope for the current (merged) pull request
-   * @returns {Promise} a promise - resolved with a String scope of the PR or rejected if no valid scope found
+   * @returns {Promise} a promise - resolved with PR info (changelog and scope) or rejected on error
    */
   _getMergedPrInfo () {
     return this._getLastPr()
       .then((pr) => {
-        return {
-          changelog: utils.getChangelogForPr(pr),
-          scope: utils.getScopeForPr(pr)
-        }
+        return Promise.all([utils.getChangelogForPr(pr), utils.getScopeForPr(pr)]).then((result) => {
```

Comment:
It looks like it does in 6.x but not 5.x
http://node.green/#destructuring--declarations

I can maybe make the new version of `pr-bumper` only work in `6.x` then I can use it. actually I like that idea. I think a couple apps are still building with 5.x node, but if `engines` is set, I don't think an `npm install -g pr-bumper` will pull in the new one if you're still using an old, incompatible node version.